flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -99,11 +99,11 @@
       "deleteNotificationChannelGroup";
   private static final String CREATE_NOTIFICATION_CHANNEL_METHOD = "createNotificationChannel";
   private static final String DELETE_NOTIFICATION_CHANNEL_METHOD = "deleteNotificationChannel";
-  private static final String GET_ACTIVE_NOTIFICATIONS_METHOD = "getActiveNotifications";
   private static final String GET_NOTIFICATION_CHANNELS_METHOD = "getNotificationChannels";
   private static final String START_FOREGROUND_SERVICE = "startForegroundService";
   private static final String STOP_FOREGROUND_SERVICE = "stopForegroundService";
   private static final String PENDING_NOTIFICATION_REQUESTS_METHOD = "pendingNotificationRequests";
+  private static final String ACTIVE_NOTIFICATIONS_METHOD = "getActiveNotifications";
```

Comment:
Forgot to mention that since the method from Dart side has been renamed back to `getActiveNotifications`, `ACTIVE_NOTIFICATIONS_METHOD` can be renamed to `GET_ACTIVE_NOTIFICATIONS_METHOD` for consistency. Similar comment applies to the iOS and macOS code